pkg/instance/instance.go

```diff
@@ -171,7 +171,9 @@ func (env *env) BuildKernel(buildCfg *BuildKernelConfig) (
 }
 
 func SetConfigImage(cfg *mgrconfig.Config, imageDir string, reliable bool) error {
-	cfg.KernelObj = filepath.Join(imageDir, "obj")
+	if cfg.Type != targets.Starnix {
+		cfg.KernelObj = filepath.Join(imageDir, "obj")
```

Comment:
It will probably take significant work to make it work correctly, but it seems like we might eventually want to have the starnix binary available for doing e.g. coverage reports... does creating an empty file (or copying over the `starnix_runner` binary as-is) cause issues currently?

Reply:
Yes, it would be better to create an empty file in "obj/", or copy real binary there.
syz-ci also knows about this layout and will upload the object file and provide in bug reports:
https://github.com/google/syzkaller/blob/89298aad70c7bd5d018a868dc31bc6ff4dca7f27/syz-ci/manager.go#L800
Also it's used for symbolization of reports, so may be useful even w/op coverage.

Reply:
Thanks, this might be possible to do now -- I'm not 100% sure which file is the right one to use as the kernel obj here, but there are only a few options. I'll try to get more info and update soon.

Reply:
I think it is `out/x64/exe.unstripped/starnix_kernel`

Reply:
Done!